README.md

```diff
@@ -1,2 +1,124 @@
 # alsf-scpca
 Management and analysis tools for ALSF Single-cell Pediatric Cancer Atlas data.
+
+- [Environment Setup](#environment-setup)
+  - [Nextflow](#nextflow)
+    - [Installing Nextflow](#installing-nextflow)
+    - [Nextflow Tower](#nextflow-tower)
+  - [Docker](#docker)
+  - [AWS](#aws)
+    - [Installing AWS Command line tools](#installing-aws-command-line-tools)
+    - [Configuring your AWS credentials](#configuring-your-aws-credentials)
+    - [AWS infrastructure](#aws-infrastructure)
+- [Running Workflows](#running-workflows)
+- [Running locally](#running-locally)
+  - [Running on AWS Batch](#running-on-aws-batch)
+
+## Environment Setup
+
```

Comment:
Could put a note at the top about this probably won't run locally. (Not sure if I needed to do this environment set up locally since it won't be able to run?)

Reply:
You do need to do the environment setup regardless; it needs to have the aws credentials to run on batch. Docker is not needed if you are not running locally, but you need everything else.